src/SamplesApp/Benchmarks.Shared/Suite/Windows_UI_Xaml/ResourceDictionaryBench/XamlControlsResourcesCreationBenchmark.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using BenchmarkDotNet.Attributes;
+using Microsoft.UI.Xaml.Controls;
+
+namespace SamplesApp.Benchmarks.Suite.Windows_UI_Xaml.ResourceDictionaryBench
+{
+	public class XamlControlsResourcesCreationBenchmark
+	{
+		[Benchmark]
+		public void Create_XamlControlsResources()
+		{
+			var xcr = new XamlControlsResources();
```

Comment:
Codacy found an issue: [Remove the unused local variable 'xcr'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6177554)